Gui/opensim/view/src/org/opensim/view/motions/MotionDisplayer.java

```diff
@@ -988,7 +987,7 @@ private void createDefaultMotionObjects() {
             UUID uuidForMarkerGeometry = UUID.randomUUID();
             experimentalMarkerGeometryJson.put("uuid", uuidForMarkerGeometry.toString());
             experimentalMarkerGeometryJson.put("type", "SphereGeometry");
-            experimentalMarkerGeometryJson.put("radius", DEFAULT_MARKER_SIZE);
+            experimentalMarkerGeometryJson.put("radius", DEFAULT_MARKER_SIZE*experimentalMarkerScaleFactor);
```

Comment:
This still looks troublesome to me. `DEFAULT_MARKER_SIZE` does not appear to be the default user-specified marker radius from the preference, but the `currentSize`. The `experimentalMarkerScaleFactor` just scales an already scaled `currentSize` which makes the user setting meaningless. @nickbianco @chrisdembia @jimmyDunne and I all advocated for not using a `experimentalMarkerScaleFactor` because it makes sizing ambiguous. So why are we doing this?

Reply:
@aseth1 DEFAULT_MARKER_SIZE is set on construction and can only be changed by updating from the user preference, while experimentalMarkerScaleFactor is the live user specified scaleup factor with initial value of 1. I believe we agreed on a plan/resolution already regarding the handling of the scale factor at this point.

Reply:
Again this is an issue with variable naming and code documentation. Without expanding the full file I could see:

```
DEFAULT_MARKER_SIZE = Double.parseDouble(currentSize);
```

Up above one sees that `currentSize` is set to default. It seems this could be easily misread to be the current size in use as the variable name otherwise indicates.

Reply:
>  I believe we agreed on a plan/resolution already regarding the handling of the scale factor at this point.

My feeling is that @jenhicks and you over-ruled the team and if we are following scrum neither of you have the authority to do that.

Reply:
Uh oh @aseth1 ... it seems you have discovered the secret cabal @aymanhab and I created to defeat the scrum [evil laugh]. Actually, I think this may just be a misunderstanding. I didn't see the timing for today's sprint on the calendar, but I am back working as usual today and happy to chat anytime via zoom.

Reply:
A simple vote will easily clarify.